go.mod

```diff
@@ -71,7 +71,7 @@ replace (
 	k8s.io/metrics => k8s.io/metrics v0.21.2
 	k8s.io/mount-utils => k8s.io/mount-utils v0.21.2
 	k8s.io/sample-apiserver => k8s.io/sample-apiserver v0.21.2
-	sigs.k8s.io/controller-runtime => sigs.k8s.io/controller-runtime v0.9.2
+	sigs.k8s.io/controller-runtime => sigs.k8s.io/controller-runtime v0.11.0
```

Comment:
I think this is misleading. We should not bump these deps. We should rather pin (= add in the `replace` section)
```
github.com/go-logr/logr => github.com/go-logr/logr v0.4.0
github.com/go-logr/zapr => github.com/go-logr/zapr v0.4.0
k8s.io/klog/v2 => k8s.io/klog/v2 v2.8.0
```